jwst/scripts/world_coords.py

```diff
@@ -1,5 +1,20 @@
 #!/usr/bin/env python
 
+"""
+Read in the output of extract2d and apply the WCS transforms to all pixels in a slit.
+
+The extract2d file can be (FS and MOS) or assign_wcs (IFU).
+
+For each slit it writes the results as a cube with four planes (wavelength, ra, dec, y_slit)
+in a separate fits extension.
+
+The file is saved with a suffix 'world_coordinates'.
+
+Requested by the NIRSPEC team.
+
+Build 7.1 testing.
```

Comment:
```suggestion
```

Reply:
I decided to re-word but leave it there to preserve history.